### ISSUE TITLE
heaptrace-git AUR dependency "gcc make"?

### ISSUE THREAD

**Issue by caprinux** [Arinerron/heaptrace#73]
Did I miss something or is the dependency broken? 

![image](https://user-images.githubusercontent.com/76640319/143792137-8a7012f5-5422-4616-85ba-d14ad75dde93.png)

**Comment by Arinerron** [comment]
Hey @caprinux, thank you for reporting this!!

Sorry about the late reply. Usually I get an email when someone files an issue, but I didn't seem to for this one for some reason.

Would you mind running `pacman -Qo /usr/bin/make /usr/bin/gcc` and sending the output? It seems that your `make`/`gcc` was provided by a different package.

**Comment by Arinerron** [comment]
I'll also try to get a binary package up in a bit so I can drop those build dependencies

**Comment by caprinux** [comment]
![image](https://user-images.githubusercontent.com/76640319/144336886-07c701e8-c894-42eb-ab30-db400d6119fd.png)

> Sorry about the late reply. Usually I get an email when someone files an issue, but I didn't seem to for this one for some reason.

No problem! Thanks for the reply :)

**Comment by Arinerron** [comment]
I wish I could reproduce the issue but I installed yay and still can't :( I have the same packages that provide those binaries. Not sure why yay doesn't see that `make` and `gcc` packages are already installed for you...
![image](https://user-images.githubusercontent.com/3926753/144338715-58c383b2-50be-4d86-bb94-993bbcd65863.png)

I read a little more about this and it turns out you're not supposed to include `make` or `gcc` in the PKGBUILD's makedepends anyway, because it's assumed that `base-devel` is assumed:
> Note: The group base-devel is assumed to be already installed when building with makepkg. Members of this group should not be included in makedepends array.

https://wiki.archlinux.org/title/PKGBUILD#makedepends

So I removed those from the latest PKGBUILD in the AUR. Hopefully this fixes your issue :) Let me know if not and I'll see what else I can do.
